manim/animation/composition.py

```diff
@@ -166,9 +160,9 @@ def __init__(
 class LaggedStartMap(LaggedStart):
     def __init__(
         self,
-        AnimationClass: Animation,
+        AnimationClass: Callable[..., Animation],
```

Comment:
Can you add to the guideline what these three dots mean?

Reply:
Not really... it's the constructor of an Animation object and it takes way too many arguments.

Reply:
okay